src/demo-app/chips/chips-demo.html

```diff
@@ -31,6 +31,47 @@ <h4>Advanced</h4>
         </mat-chip>
       </mat-chip-list>
       <div>{{message}}</div>
+
+      <h4>With avatar and icons</h4>
+
+
+      <mat-chip-list>
+        <mat-chip>
+          <mat-icon matChipAvatar>home</mat-icon>
+          Home
+          <mat-icon matChipRemove matChipTrailingIcon>cancel</mat-icon>
```

Comment:
what does `matChipTrailingIcon` do? it looks the same as the example above where you only applied `matChipRemove`

Reply:
Ah ok, do these examples need both directives? It seems like just matChipRemove would be sufficient since does everything that trailing icon does + some extra behavior. In fact you could probably make the CSS simpler by having `matChipRemove` just apply the `mat-chip-trailing-icon` class .

Also should add an example of a trailing icon that isn't a remove icon